include/atomicassets.hpp

```diff
@@ -370,6 +406,16 @@ CONTRACT atomicassets : public contract {
     config_t       config       = config_t(get_self(), get_self().value);
     tokenconfigs_t tokenconfigs = tokenconfigs_t(get_self(), get_self().value);
 
+    void create_template(
+        name & authorized_creator,
+        name & collection_name,
+        name & schema_name,
+        bool transferable,
+        bool burnable,
+        uint32_t max_supply,
+        ATTRIBUTE_MAP & immutable_data,
+        ATTRIBUTE_MAP mutable_data = {}
```

Comment:
There's no reference for the mutable_data because the original createtempl action does not have a field for mutable_data (only immutable_data). It's only accessed by the new createtempl2 action.